lib/cylc/cfgspec/gcylc.py

```diff
@@ -29,48 +31,56 @@
 
 "gcylc config file format."
 
-SITE_FILE = os.path.join( os.environ['CYLC_DIR'], 'conf', 'gcylcrc', 'themes.rc' )
-USER_FILE = os.path.join( os.environ['HOME'], '.cylc', 'gcylc.rc' )
+SITE_FILE = os.path.join(
+    os.environ['CYLC_DIR'], 'conf', 'gcylcrc', 'themes.rc')
+USER_FILE = os.path.join(os.environ['HOME'], '.cylc', 'gcylc.rc')
 
 SPEC = {
-    'initial views'   : vdr( vtype='string_list', default=["text"] ),
-    'ungrouped views' : vdr( vtype='string_list', default=[] ),
-    'use theme'       : vdr( vtype='string', default="default" ),
-    'dot icon size'   : vdr( vtype='string', default="medium",
-                             options=["small","medium","large", "extra large"]),
-    'sort by definition order' : vdr( vtype='boolean', default=True ), 
-    'task filter highlight color' : vdr(vtype='string', default='PowderBlue'),
-    'initial side-by-side views' : vdr(vtype='boolean', default=False),
-    'themes' : {
-        '__MANY__' : {
-            'inherit'       : vdr( vtype='string', default="default" ),
-            'defaults'      : vdr( vtype='string_list' ),
-            'waiting'       : vdr( vtype='string_list' ),
-            'held'          : vdr( vtype='string_list' ),
-            'queued'        : vdr( vtype='string_list' ),
-            'ready'         : vdr( vtype='string_list' ),
-            'expired'       : vdr( vtype='string_list' ),
-            'submitted'     : vdr( vtype='string_list' ),
-            'submit-failed' : vdr( vtype='string_list' ),
-            'running'       : vdr( vtype='string_list' ),
-            'succeeded'     : vdr( vtype='string_list' ),
-            'failed'        : vdr( vtype='string_list' ),
-            'retrying'      : vdr( vtype='string_list' ),
-            'submit-retrying' : vdr( vtype='string_list' ),
-            'runahead' : vdr( vtype='string_list' ),
-            },
+    'initial views': vdr(vtype='string_list', default=["text"]),
+    'ungrouped views': vdr(vtype='string_list', default=[]),
+    'use theme': vdr(vtype='string', default="default"),
+    'dot icon size': vdr(
+        vtype='string',
+        default="medium",
+        options=["small", "medium", "large", "extra large"]),
+    'sort by definition order': vdr(vtype='boolean', default=True),
+    'task filter highlight color': vdr(vtype='string', default='PowderBlue'),
+    'initial side-by-side views': vdr(vtype='boolean', default=False),
+    'themes': {
+        '__MANY__': {
+            'inherit': vdr(vtype='string', default="default"),
```

Comment:
Previous version was more readable, but I guess PEP8 is telling you to remove the extra spaces between key and value?